shell/platform/android/test/io/flutter/embedding/android/FlutterAndroidComponentTest.java

```diff
@@ -81,7 +81,7 @@ public void pluginsReceiveFlutterPluginBinding() {
     assertNotNull(binding.getTextureRegistry());
     assertNotNull(binding.getPlatformViewRegistry());
 
-    delegate.onActivityCreated(null);
+    delegate.onRestoreInstanceState(null);
     delegate.onCreateView(null, null, null);
```

Comment:
It's worth pointing out that this was a sign of the bug, as this is not the correct lifecycle order per https://developer.android.com/guide/components/fragments#Creating